package/MDAnalysis/analysis/distances.py

```diff
@@ -55,6 +54,9 @@ def contact_matrix(coord, cutoff=15.0, returntype="numpy", box=None):
     .. versionchanged:: 0.11.0
        Keyword *suppress_progmet* and *progress_meter_freq* were removed.
     '''
+    
+    from scipy import sparse      
```

Comment:
We decided in #666 to put *all* imports up front and don't hide any imports inside methods any more. (Just realized that I need to add this to the Style Guide...)

If this change makes tests fail in the minimal environment then these tests have to be marked appropriately... ask @mnmelo if you want to find out how ;-)